ginivision/src/doc/source/guide-for-open-with.rst

```diff
@@ -0,0 +1,182 @@
+How to Upload PDFs and Images
+=============================
+
+General considerations
+----------------------
+
+Enabling your app to open PDFs and images allows your users to open any kind of files which are identified by the OS as PDFs or images. Make sure to check the MIME type and that the file size is below an acceptable threshold (5MB for example). It is also advisable to check the first bytes of the incoming files to determine their type and allow only PDFs and known image types.
```

Comment:
Please see comment in iOS version: if we only want to allow single-page documents and a check for this is possible, please add it as a suggested check under General considerations.